crates/hotshot/examples/infra/mod.rs

```diff
@@ -884,16 +884,12 @@ pub async fn main_entry_point<
 
     run_config.node_index = node_index.into();
 
-    let (public_key, private_key) =
-        <<TYPES as NodeType>::SignatureKey as SignatureKey>::generated_from_seed_indexed(
+    run_config.config.my_own_validator_config =
+        ValidatorConfig::<<TYPES as NodeType>::SignatureKey>::generated_from_seed_indexed(
             run_config.seed,
             node_index.into(),
+            1,
```

Comment:
nit: `stake_value: 1` would be clearer

Reply:
This is a function call, cannot do this.